### PR TITLE
Front end fixes

### PR DESCRIPTION
**Pull request by ahmedzaky030**
@norhan22 the pull request cannot be merged ,  please git pull and solve confilcts and push again

### CODE REVIEW

**Comment by ahmedzaky030** [comment]
@norhan22  you can check the confilcts files mentioned above

File: app/index.html [hunk 1]
```diff
@@ -76,9 +76,9 @@
         // ga('send', 'pageview');
     </script>
     <script src="https://maps.googleapis.com/maps/api/js?libraries=places"></script>
-    <!-- <script src="http://localhost:8080/socket.io/socket.io.js"></script> -->
+    <script src="http://localhost:8080/socket.io/socket.io.js"></script>
```
Comment (ahmedzaky030) [inline app/index.html]: Please don't push any uncommented local urls

File: app/index.html [hunk 2]
```diff
@@ -76,9 +76,9 @@
...
 
-    <script src="http://18.216.100.201:8080/socket.io/socket.io.js"></script>
+    <!-- <script src="http://18.216.100.201:8080/socket.io/socket.io.js"></script> -->
```
Comment (ahmedzaky030) [inline app/index.html]: only online urls are pushed to github

File: app/index.html [hunk 3]
```diff
@@ -152,7 +152,8 @@
     <script src="scripts/services/publicSpeakingService.js"></script>
     <script src="scripts/services/quizFactory.js"></script>
     <script src="scripts/services/chatService.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-validator/0.4.5/js/bootstrapvalidator.min.js" />
+    <!-- <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-validator/0.4.5/js/bootstrapvalidator.min.js" /> -->
```
Comment (ahmedzaky030) [inline app/index.html]: I think it's removed please check it again @norhan22

File: app/index.html (hunk 1 above)
Comment (norhan22) [reply app/index.html]: I will do it and push the changes and sorry for this conflict

**Pull request closed without merging**